azure/iac/sql_servers.rego

```diff
@@ -28,9 +44,9 @@ sql_public_access_err = "SQL servers with public access detected!" {
 }
```

Comment:
message should be: `Public Network Access is currently not disabled on the resource`